src/index.html

```diff
@@ -11,6 +11,36 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="#"
+    class="form form-large"
+    data-qa="big keypress"
+    >
+      <img class="search-icon--large"
+      src="images/Search.svg"
+      alt="search-icon"
+      >
+
+      <input class="searchbar searchbar-large"
```

Comment:
`searchbar-large` thats naming for a block, here you make it like a modifier, so name it like at the prev img tag